### DISCUSSION TITLE
Drescher und Häcksler wird nicht abgetankt, wenn man selber fährt.

### DISCUSSION THREAD

**Discussion by Shifty263** [Courseplay/courseplay#6963] [Q&A] [answered]
Hallo,

ich war sehr lange auf einer alten CP Version unterwegs, nun hatte ich mich entschieden dem CP mal wieder ein Update zu Spendieren.

Was mir sofort auffiel, da ich den CP hauptsächlich als Abfahrer benutze, das ich wenn ich den Drescher bzw den Häcksler selbst fahre nicht mehr Abgetankt werde. 

Der CP ist im Mode2 kommt aber nicht zum Drescher bzw Häcksler wenn ich den Drescher bzw Häcksler selbst Fahre.

Wenn der Drescher bzw der Häcksler ebenfalls vom CP gefahren geht es allerdings.

Meine Frage ist nun ob ich da etwas einstellen muss damit mich der CP wieder erkennt oder wurde diese möglichkeit den Drescher bzw Häcksler selbst zu fahren und sich nur abfahren zu lassen generell heraus genommen?

Danke im voraus

**Comment by TinyCrocodile** [comment, marked as answer]
Wurde generell raus genommen. Mode 2 nur mit cp  ernter.

**Reply by Shifty263** [reply]
Schade. Kann man das irgendwie selbst wieder einfügen bzw aktivieren?

**Reply by Tensuko** [reply]
Nein, kann man nicht. Das System arbeitet komplett anders.
Für deinen Fall, musst du dir dann mal AutoDrive anschauen.
